recipe/build.sh

```diff
@@ -9,9 +9,25 @@ if [[ $target_platform == "linux-"* ]]; then
     export LIBS="-lrt $LIBS"
 fi
 
-./configure \
-    --enable-shared \
-    --prefix=$PREFIX
+BUILD_ARCH=${CONDA_TOOLCHAIN_BUILD%%-*}
+HOST_ARCH=${CONDA_TOOLCHAIN_HOST%%-*}
 
-make -j${CPU_COUNT} check
-make install
\ No newline at end of file
+configure_cmd=(./configure --enable-shared --prefix=$PREFIX)
+
+# cross-compiling needs to use prebuilt capnp
+# https://github.com/capnproto/capnproto/issues/1815#issuecomment-1732327995
+if [[ "${BUILD_ARCH}" != "${HOST_ARCH}" ]]; then
+    mamba install --no-deps --yes capnproto
```

Comment:
You should add this to `build: - capnproto # [build_platform != target_paltform]`

Reply:
I tried but installation fails because of openssl incompatibility with other required packages

Reply:
Maybe would be fixed by https://github.com/conda-forge/capnproto-feedstock/pull/32